### PR TITLE
Update Initial Framework

### PR DESCRIPTION
**Pull request by wsciaroni**
Use opensource library to start over...

### CODE REVIEW

File: android/app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -1,47 +1,54 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="com.example.myportion_app">
+    package="io.instamobile.myportion_app">
```
Comment (CollierA13j) [inline android/app/src/main/AndroidManifest.xml]: change to "com.example.myportion_app"

File: android/app/src/main/kotlin/io/instamobile/flutter_login_screen/MainActivity.kt [hunk 2]
```diff
@@ -0,0 +1,12 @@
+package io.instamobile.myportion_app
```
Comment (CollierA13j) [inline android/app/src/main/kotlin/io/instamobile/flutter_login_screen/MainActivity.kt]: remove instamobile package

File: android/app/src/profile/AndroidManifest.xml [hunk 3]
```diff
@@ -1,7 +1,7 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="com.example.myportion_app">
+    package="io.instamobile.myportion_app">
```
Comment (CollierA13j) [inline android/app/src/profile/AndroidManifest.xml]: change to "com.example.myportion_app"

File: ios/Runner.xcodeproj/project.pbxproj [hunk 4]
```diff
@@ -431,7 +454,7 @@
 					"$(inherited)",
 					"$(PROJECT_DIR)/Flutter",
 				);
-				PRODUCT_BUNDLE_IDENTIFIER = com.example.myportionApp;
+				PRODUCT_BUNDLE_IDENTIFIER = io.instamobile.login.ios;
```
Comment (CollierA13j) [inline ios/Runner.xcodeproj/project.pbxproj:457]: remove instamobile

File: ios/Runner.xcodeproj/project.pbxproj [hunk 5]
```diff
@@ -458,7 +481,7 @@
...
```
Comment (CollierA13j) [inline ios/Runner.xcodeproj/project.pbxproj:484]: remove instamobile

File: ios/Runner.xcodeproj/project.pbxproj [hunk 6]
```diff
@@ -299,7 +320,7 @@
...
```
Comment (GuassianFlux) [inline ios/Runner.xcodeproj/project.pbxproj:323]: Instamobile here

File: android/app/src/debug/AndroidManifest.xml [hunk 7]
```diff
@@ -1,7 +1,7 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="com.example.myportion_app">
+    package="io.instamobile.myportion_app">
```
Comment (CollierA13j) [inline android/app/src/debug/AndroidManifest.xml]: change to "com.example.myportion_app"

File: ios/Runner.xcodeproj/project.pbxproj (hunk 4 above)
Comment (GuassianFlux) [inline ios/Runner.xcodeproj/project.pbxproj:457]: Instamobile here

File: ios/Runner.xcodeproj/project.pbxproj (hunk 6 above)
Comment (CollierA13j) [inline ios/Runner.xcodeproj/project.pbxproj:323]: remove instamobile

File: ios/Runner.xcodeproj/project.pbxproj (hunk 5 above)
Comment (GuassianFlux) [inline ios/Runner.xcodeproj/project.pbxproj:484]: Instamobile here

File: ios/Runner.xcodeproj/project.pbxproj (hunk 4 above)
Comment (wsciaroni) [reply ios/Runner.xcodeproj/project.pbxproj:457]: Will Not Fix

File: ios/Runner.xcodeproj/project.pbxproj (hunk 5 above)
Comment (wsciaroni) [reply ios/Runner.xcodeproj/project.pbxproj:484]: Will Not Fix

File: ios/Runner.xcodeproj/project.pbxproj (hunk 6 above)
Comment (wsciaroni) [reply ios/Runner.xcodeproj/project.pbxproj:323]: Will Not Fix

File: ios/Runner.xcodeproj/project.pbxproj (hunk 4 above)
Comment (wsciaroni) [reply ios/Runner.xcodeproj/project.pbxproj:457]: Will Not Fix

File: ios/Runner.xcodeproj/project.pbxproj (hunk 6 above)
Comment (wsciaroni) [reply ios/Runner.xcodeproj/project.pbxproj:323]: Will Not Fix

File: ios/Runner.xcodeproj/project.pbxproj (hunk 5 above)
Comment (wsciaroni) [reply ios/Runner.xcodeproj/project.pbxproj:484]: Will Not Fix

**Review by CollierA13j — COMMENTED** [review]
Looks good for now

**Review by GuassianFlux — APPROVED** [review]

**Review by CollierA13j — APPROVED** [review]
Approved

**Comment by wsciaroni** [comment]
## Background

Merge in code so that we all have a common starting place.

## NOTE

This code is preliminary.  WE WOULD NOT NORMALLY MERGE THIS MUCH UNRELATED  MATERIAL

**Pull request merged**